examples/neg/boolean_wrong_name.effekt

```diff
@@ -0,0 +1,3 @@
+module boolean_wrong_name
+
+def alwaysTrue(): Boolean = true
```

Comment:
Once we have it for terms, we can add a hint `True` ~> `true` for @BinderDavid ;)

And `S(S(Z()))` ~> "Did you mean `2`?"